systemd/systemd.go

```diff
@@ -413,6 +413,7 @@ WantedBy={{.ServiceSystemdTarget}}
 		"",
 		desc.SocketFileName,
 		restartCond,
+		desc.Type,
```

Comment:
At some point in the future (not this branch) we should replace that struct by a properly typed struct defined at the global level, and prefix all these values with their respective field names.